lnrpc/signrpc/signer_server.go

```diff
@@ -34,6 +35,9 @@ const (
 	// SubServerConfigDispatcher instance recognize this as the name of the
 	// config file that we need.
 	subServerName = "SignRPC"
+
+	// BIP0340 is the prefix for BIP0340-related tagged hashes.
+	BIP0340 = "BIP0340"
```

Comment:
I want to add `TapSighash` here as well to the disallow list. However, I'm not sure if there are valid use cases where a user invokes `SignMessage` w/ a certain `KeyLocator` to send funds?

Reply:
Good point! I think `SignMessage` should _never_ be used for anything transaction related, so `TapSighash` should be disallowed as well IMO.